cmake/org.eclipse.cdt.cmake.ui/src/org/eclipse/cdt/cmake/ui/internal/CMakeBuildTab.java

```diff
@@ -37,16 +39,17 @@ public class CMakeBuildTab extends CommonBuildTab {
 	 * Checkbox allowing user to choose these settings over the default operating system defaults.
```

Comment:
Comment should be updated to something like:

	/**
	 * Checkbox, controls whether to use UI specified values or defaults set in
	 * {@link CMakeBuildConfiguration#getDefaultProperties()}.
	 * See {@link CMakeBuildConfiguration#CMAKE_USE_DEFAULT_CMAKE_SETTINGS}
	 */

Reply:
I decided this comment added little value and removed it. i.e. all the fields connect to their given field in ICMakeBuildConfiguration and leaving the code to document itself is better than repeating it. This is true because this is a private field, if it was API having javadoc would be more useful.